components/dom_crawler.rst

```diff
@@ -195,6 +204,14 @@ Get all the direct child nodes matching a CSS selector::
 
     $crawler->filter('body')->children('p.lorem');
 
+.. versionadded:: 4.4
```

Comment:
Lets move it below the code example please

Reply:
I'm not sure to understand. Is dc1e6b550 ok ?